tests/test_symbolics.py

```diff
@@ -705,7 +704,7 @@ def test_multibounds_op(self):
         op.apply(time_M=5)
         fnorm2 = norm(f)
 
-        assert fnorm == fnorm2
+        assert np.isclose(fnorm, fnorm2, rtol=np.finfo(fnorm.dtype).eps)
```

Comment:
lolz, I struggle to understand this test at all... aren't those two Operators equivalent?

and if, again, should we not just expect perfect equality

Reply:
No idea